common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -444,3 +446,18 @@ def test_connected_accounts(self):
         for field_id, title, link_title in providers:
             self.assertEqual(self.account_settings_page.title_for_field(field_id), title)
             self.assertEqual(self.account_settings_page.link_title_for_link_field(field_id), link_title)
+
+
+@attr('a11y')
+class AccountSettingsA11yTest(AccountSettingsTestMixin, WebAppTest):
+    """
+    Class to test account settings accessibility.
+    """
+
+    def test_account_settings_axs(self):
```

Comment:
Could you update the test case names too?

`test_account_settings_axs` --> `test_account_settings_a11y`

(Same for the test cases in LearnerProfileA11yTest.)